core/comments/rendered_workspace_comment.ts

```diff
@@ -208,8 +208,14 @@ export class RenderedWorkspaceComment
   private startGesture(e: PointerEvent) {
     const gesture = this.workspace.getGesture(e);
     if (gesture) {
-      gesture.handleCommentStart(e, this);
-      this.workspace.getLayerManager()?.append(this, layers.BLOCK);
+      if (browserEvents.isTargetInput(e)) {
+        // If the text area was the focus, don't allow this event to bubble up
+        // and steal focus away from the editor/comment.
+        e.stopPropagation();
+      } else {
+        gesture.handleCommentStart(e, this);
```

Comment:
Next question: why is this not already fixed by the other uses of `isTargetInput` in `inject.ts` and `gesture.ts`?

Reply:
`handleCommentStart()` doesn't ever call `doStart()`, where `isTargetInput()` is checked; `handleWsStart` does, but by the time that's called the event has already bubbled up to the workspace and stolen focus away from the textarea, so it's too late. Likewise, inject.ts only checks it for keydown and contextmenu events, not clicks.